react-forms/src/utils/classnames.ts

```diff
@@ -0,0 +1,46 @@
+type ImportedStyles = CSSModule | { [cs: string]: string }
```

Comment:
I saw the same code in other repository of yours.
Perhaps you want to create a custom npm package (public) and use it in your projects. It's a suggestion.